src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -204,8 +204,12 @@ public function getBundles()
     /**
      * {@inheritdoc}
      */
-    public function getBundle($name, $first = true)
+    public function getBundle($name, $first = true, $noDeprecation = false)
```

Comment:
signature change should be removed, `func_get_arg` should be used instead.

Reply:
fixed